DB/Item.lua

```diff
@@ -150,4 +150,17 @@ function Item:IsMount(entry)
 	return entry.type == CONSTANTS.ITEM_TYPES.MOUNT
 end
 
+function Item:HasZone(entry)
+	if entry.coords then
+		for _, value in pairs(entry.coords) do
```

Comment:
What does the ``value`` represent? Unless it's a generic table function, I'd use a name that accurately describes the type of object we're iterating over. In this case, probably something like ``waypoint`` or ``waypointData``?

Reply:
Sure. I've renamed it in 039729e9df6633a37a838bf692b3abb8955a5b95.